projects/ama-sdk/src/lib/helpers/common.ts

```diff
@@ -21,6 +21,7 @@ export interface EntityDialogForm {
     name: string;
     description?: string;
     projectId?: string;
+    template?: string;
```

Comment:
Why is it here? EntityDialogForm doesn't have a template. I think you may mix something together, but this is not the right way to do it, I assume.